static/locales/en-US.yaml

```diff
@@ -512,6 +512,8 @@ Channel:
   Your search results have returned 0 results: Your search results have returned 0
     results
   Sort By: Sort By
+  Shorts:
+    Shorts: Shorts
```

Comment:
Add `This channel does not currently have any shorts` below

Reply:
✅